crates/node/src/storage/rocksdb.rs

```diff
@@ -737,6 +737,72 @@ impl RocksDB {
             .get_cf(rollback_cf, key)
             .map_err(|e| Error::DBError(e.into_string()))
     }
+
+    /// Writes an entry directly to the db
+    pub fn insert_entry(
```

Comment:
Should we just rm the `height` param since it's not implemented? We could maybe export and use the existing `add_value_bytes_to_batch` instead (also the doc line here is not accurate)

Reply:
sure